index.js

```diff
@@ -43,7 +43,7 @@ async function run () {
     const envFileName = core.getInput('environment-file')
     const envFilePath = path.join(process.env.GITHUB_WORKSPACE || '', envFileName)
     const envYaml = yaml.safeLoad(fs.readFileSync(envFilePath, 'utf8'))
-    const envName = envYaml.name
+    const envName = core.getInput('environment-name') || envYaml.name
```

Comment:
This var is only used for outputting. You'll have to pass it to micromamba to get the right env name there too.

Reply:
My bad, it should be ok with the latest commit